src/test/scala/com/ibm/couchdb/api/DocumentsSpec.scala

```diff
@@ -16,7 +16,7 @@
 
 package com.ibm.couchdb.api
 
-import com.ibm.couchdb.CouchDoc
```

Comment:
maybe just `import com.ibm.couchdb.{CouchDoc, Req}` instead of `import com.ibm.couchdb._`